### PR TITLE
Add section with migration methods explanation

### PR DESCRIPTION
**Pull request by Jeckerson**

### CODE REVIEW

File: en/db-migrations.md [hunk 1]
```diff
@@ -119,6 +119,33 @@ $migration::run([
 ]);
 ```
 
+## Migration methods
+
+Each migration is an separate class that works as entity for specific database table.
```
Comment (niden) [inline en/db-migrations.md]: as _an_ entity

**Review by sergeyklay — APPROVED** [review]

**Review by ruudboon — APPROVED** [review]

File: en/db-migrations.md [hunk 2]
```diff
@@ -119,6 +119,33 @@ $migration::run([
...
+Inside each class there are different methods that can occur during migration running.
+
+You can also execute whatever logic you want inside each migration, without any limitation.
```
Comment (niden) [inline en/db-migrations.md]: Each migration file (and class) can implement specific methods, that will be executed based on the operation requested. There are no restrictions on the logic encapsulated in each method.

File: en/db-migrations.md [hunk 3]
```diff
@@ -119,6 +119,33 @@ $migration::run([
...
+
+Here are tables with Migration Class methods. Table is sorted by order of execution: from earliest to latest.
```
Comment (niden) [inline en/db-migrations.md]: What do you mean by _here are tables...._?

**Review by niden — CHANGES_REQUESTED** [review]

File: en/db-migrations.md (hunk 3 above)
Comment (Jeckerson) [reply en/db-migrations.md]: 2 HTML tables below

File: en/db-migrations.md [hunk 4]
```diff
@@ -119,6 +119,34 @@ $migration::run([
...
 
+## Migration methods
+
+Each migration is an separate class that works as an entity for specific database table.
+Inside each class there are different methods that can occur during migration running.
+
+Each migration file (and class) can implement specific methods, that will be executed based on the operation requested.
+There are no restrictions on the logic encapsulated in each method.
+
+Here are tables with Migration Class methods. Table is sorted by order of execution: from earliest to latest.
```
Comment (niden) [inline en/db-migrations.md]: The tables below show the Migration Class methods. They are stored by order of execution, earliest to latest.

**Review by niden — CHANGES_REQUESTED** [review]

**Review by niden — APPROVED** [review]

**Pull request merged**